src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -9,7 +9,9 @@ export const GithubCreateBranchOptionsGet = async (req: Request, res: Response,
 	const { issue_key: key } = req.query;
 
 	if (!jiraHost) {
-		return next(new Error(Errors.MISSING_JIRA_HOST));
+		req.log.warn({ req, res }, Errors.MISSING_JIRA_HOST);
+		res.status(404).send(Errors.MISSING_JIRA_HOST);
```

Comment:
I think this should be 400- bad request, not 404 - page not found...

Reply:
Updated to 400